### PR TITLE
Remove Namespace Annotator

### PR DESCRIPTION
**Pull request by alecmerdler**
### Description

It's no longer being used. The UI was updated in https://github.com/openshift/console/pull/776.

### CODE REVIEW

File: pkg/controller/operators/olm/operator_test.go [hunk 1]
```diff
@@ -141,7 +139,6 @@ func NewFakeOperator(clientObjs []runtime.Object, k8sObjs []runtime.Object, extO
 		lister:    operatorlister.NewLister(),
 		resolver:  resolver,
 		csvQueue:  workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "clusterserviceversions"),
-		annotator: namespaceAnnotator,
 		recorder:  eventRecorder,
 		cleanupFunc: func() {
```
Comment (njhale) [inline pkg/controller/operators/olm/operator_test.go]: The cleanup func needs to go too.

**Comment by ecordell** [comment]
see: #563

**Pull request closed without merging**